src/app/payment-hub/transactions/incoming/incoming-transactions.component.ts

```diff
@@ -246,6 +247,17 @@ export class IncomingTransactionsComponent implements OnInit, AfterViewInit {
       )
       .subscribe();
 
+
```

Comment:
think the requirement was to start searching after the 4th character, so was expecting we check if length of the filter values is greater or equal to 4 before we start searching.